ddtrace/contrib/langchain/patch.py

```diff
@@ -145,6 +145,30 @@ def _tag_openai_token_usage(
         _tag_openai_token_usage(span._parent, llm_output, propagated_cost=propagated_cost + total_cost, propagate=True)
 
 
+def _is_openai_llm_instance(instance):
+    """Safely check if a traced instance is an OpenAI LLM.
+    langchain_community does not automatically import submodules which may result in AttributeErrors.
```

Comment:
My understanding is that langchain_community deprecated the chat_models submodule, not necessarily that they do not automatically import submodules.

Reply:
From what I understand, `langchain_community` hasn't deprecated the entire `chat_models` submodule but is rather deprecating individual classes (ex: `ChatOpenAI`) as they work on setting up partner libraries (ex: `langchain_openai`). Also, `langchain_community.__init__.py` does not automatically import its submodules, i.e.

```python
>>> import langchain_community
>>> langchain_community.llms  # try referencing directly, results in error
AttributeError: module 'langchain_community' has no attribute 'chat_models'
>>> from langchain_community import llms  # works
```